WalletWasabi/Services/WalletService.cs

```diff
@@ -806,16 +806,12 @@ public async Task<Block> GetOrDownloadBlockAsync(uint256 hash, CancellationToken
 
 							Block blockFromLocalNode = null;
 							// Should timeout faster. Not sure if it should ever fail though. Maybe let's keep like this later for remote node connection.
-							using (var cts = new CancellationTokenSource(TimeSpan.FromSeconds(64))) // 1/2 ADSL	512 kbit/s	00:00:32
+							using (var cts = new CancellationTokenSource(TimeSpan.FromSeconds(64)))
```

Comment:
Ideally this should use `RuntimeParams.Instance.NetworkNodeTimeout`, but tests don't like it.

Reply:
The NetworkNodeTimeout dynamically adjusts based on download from peers. If we'd have it for the local node, then that'd screw the adjustment, so it's good like this.